packages/design-system/src/components/button/stories/index.js

```diff
@@ -103,24 +121,7 @@ const ButtonCombosToDisplay = () => (
     </Headline>
     <Row>
       <div>
-        <Button
-          href=""
-          type={select(
-            'link as button - type',
-            Object.values(BUTTON_TYPES),
-            BUTTON_TYPES.PRIMARY
-          )}
-          variant={select(
-            'link as button - variant',
-            Object.values(BUTTON_VARIANTS),
-            BUTTON_VARIANTS.RECTANGLE
-          )}
-          size={select(
-            'link as button - size',
-            Object.values(BUTTON_SIZES),
-            BUTTON_SIZES.MEDIUM
-          )}
-        >
+        <Button href="" {...args}>
```

Comment:
💭 The button stories (light and dark) already show all the buttons in a grid. Changing these options seems to not do anything for any of those buttons _except_ the buttons at the bottom.

I think it would make a lot of sense to split those stories out into one with interactive controls and this plain one.